## Citation

Pleace contact author Tang Ruixiang (Email 296554555@qq.com / Wechat trx-robin)

## Read trimming

**trimmomatic**
> **Trimmomatic: A flexible trimmer for Illumina sequence data.** <br/>
> Bolger AM, Lohse M, Usadel B. <br/>
> Bioinformatics. 2014. <br/>
> [doi:10.1093/bioinformatics/btu170](https://doi.org/10.1093/bioinformatics/btu170)

## Read mapping, duplication removal, and quality score recalibration

**bwa mem**
> **Fast and accurate short read alignment with Burrows-Wheeler transform.** <br/>
> Li H, Durbin R. <br/>
> Bioinformatics. 2009. <br/>
> [doi:10.1093/bioinformatics/btp324](https://doi.org/10.1093/bioinformatics/btp324)

**Picard MarkDuplicates**
> **Picard toolkit.** <br/>
> Broad Institute; 2018.<br/>
> GitHub repository, online: http://broadinstitute.github.io/picard/

**GATK BaseRecalibrator**
> **The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.** <br/>
> McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. <br/>
> Genome Res. 2010. <br/>
> [doi:10.1101/GR.107524.110](https://doi.org/10.1101/GR.107524.110)

**samtool merge**
> **The Sequence Alignment/Map format and SAMtools.** <br/>
> Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. <br/>
> Bioinformatics. 2009. <br/>
> [doi:10.1093/bioinformatics/btp352](https://doi.org/10.1093/bioinformatics/btp352)

## Variant calling, genotyping, and filtering

**GATK HaplotypeCaller**, **GATK SelectVariants**, **GATK VariantFiltration**, **GATK VariantRecalibrator**
> **The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data.** <br/>
> McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. <br/>
> Genome Res. 2010. <br/>
> [doi:10.1101/GR.107524.110](https://doi.org/10.1101/GR.107524.110)

## Quality control, statistics, SNP annotation, reporting

**FastQC**
> **FastQC: a quality control tool for high throughput sequence data.** <br/>
> Andrews S. <br/>
> Babraham Bioinformatics, Babraham Institute, Cambridge, United Kingdom; 2010. <br/>
> Online: https://www.bioinformatics.babraham.ac.uk/projects/fastqc 

**samtool stats** and **samtool flagstat**
> **The Sequence Alignment/Map format and SAMtools.** <br/>
> Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. <br/>
> Bioinformatics. 2009. <br/>
> [doi:10.1093/bioinformatics/btp352](https://doi.org/10.1093/bioinformatics/btp352)

**QualiMap**
> **Qualimap 2: Advanced multi-sample quality control for high-throughput sequencing data.** <br/>
> Okonechnikov K, Conesa A, García-Alcalde F. <br/>
> Bioinformatics. 2016. <br/>
> [doi:10.1093/bioinformatics/btv566](https://doi.org/10.1093/bioinformatics/btv566)

**Picard CollectMultipleMetrics**
> **Picard toolkit.** <br/>
> Broad Institute; 2018.<br/>
> GitHub repository, online: http://broadinstitute.github.io/picard/

**VEP**
> **The Ensembl Variant Effect Predictor.** <br/>
> McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, Flicek P, Cunningham F. <br/>
> Genome Biology. 2016. <br/>
> [doi:10.1186/s13059-016-0974-4](https://doi.org/10.1186/s13059-016-0974-4)

**SeqKit**
> **SeqKit: A Cross-Platform and Ultrafast Toolkit for FASTA/Q File Manipulation.** <br/>
> Shen W, Le S, Li Y, Hu F. <br/>
> PLOS ONE 11(10), e0163962. 2016. <br/> 
> [doi:10.1371/journal.pone.0163962](https://doi.org/10.1371/journal.pone.0163962)

**bcftools stats**
> **A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data.** <br/>
> Li H. <br/>
> Bioinformatics. 2011. <br/>
> [doi:10.1093/bioinformatics/btr509](https://doi.org/10.1093/bioinformatics/btr509)

**MultiQC**
> **MultiQC: Summarize analysis results for multiple tools and samples in a single report.** <br/>
> Ewels P, Magnusson M, Lundin S, Käller M. <br/>
> Bioinformatics. 2016. <br/>
> [doi:10.1093/bioinformatics/btw354](https://doi.org/10.1093/bioinformatics/btw354)

## Additional references

**Snakemake**
> **Snakemake--a scalable bioinformatics workflow engine.** <br/>
> Köster J, Rahmann S. <br/>
> Bioinformatics. 2012. <br/>
> [doi:10.1093/bioinformatics/bts480](https://doi.org/10.1093/bioinformatics/bts480)

> **Sustainable data analysis with Snakemake.** <br/>
> Mölder F, Jablonski KP, Letcher B, Hall MB, Tomkins-Tinch CH, Sochat V, Forster J, Lee S, Twardziok SO, Kanitz A, Wilm A, Holtgrewe M, Rahmann S, Nahnsen S, Köster J. <br/>
> F1000Res 10, 33. 2021. <br/>
> [doi:10.12688/f1000research.29032.2](https://doi.org/10.12688/f1000research.29032.2)

**Bioconda**
> **Bioconda: A sustainable and comprehensive software distribution for the life sciences.** <br/>
> Grüning B, Dale R, Sjödin A, Chapman BA, Rowe J, Tomkins-Tinch CH, et al. <br/>
> Nat Methods. 2018. <br/>
> [doi:10.1038/s41592-018-0046-7](https://doi.org/10.1038/s41592-018-0046-7)

**Fastq file format**
> **The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants.** <br/>
> Cock PJA, Fields CJ, Goto N, Heuer ML, Rice PM. <br/>
> Nucleic Acids Res. 2009. <br/>
> [doi:10.1093/nar/gkp1137](https://doi.org/10.1093/nar/gkp1137)

**Fasta file format**
> **Improved tools for biological sequence comparison.** <br/>
> Pearson WR, Lipman DJ. <br/>
> Proceedings of the National Academy of Sciences. 1988. <br/>
> [doi:10.1073/pnas.85.8.2444](https://doi.org/10.1073/pnas.85.8.2444)

**SAM/BAM file format**
> **The Sequence Alignment/Map format and SAMtools.** <br/>
> Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup. <br/>
> Bioinformatics. 2009. <br/>
> [doi:10.1093/bioinformatics/btp352](https://doi.org/10.1093/bioinformatics/btp352)

**VCF file format**
> **The variant call format and VCFtools.** <br/>
> Danecek P, Auton A, Abecasis G, Albers CA, Banks E, DePristo MA, et al. <br/>
> Bioinformatics. 2011.<br/>
> [doi:10.1093/bioinformatics/btr330](https://doi.org/10.1093/bioinformatics/btr330)
